store/src/sql/mod.rs

```diff
@@ -1,14 +1,17 @@
+mod event;
```

Comment:
I thought we standardized on `sql.rs` rather then `/sql/mod.rs` that the `/folder/mod.rs` pattern was the old way of laying out code?

Reply:
uhh.. maybe? I like the folder/mod.rs to control the exports, but I guess I can make store/src/sql.rs and it will be the same as the store/src/sql/mod.rs file. I kind of like keeping the module exports closer to the folder structure but I'll do whatever, especially if that's the preference